pkg/aws/elb.go

```diff
@@ -14,6 +17,9 @@ type LoadBalancer struct {
 	CanonicalZoneID string
 }
 
+//LoadBalancerExtractor is a func type which is extract Classic and Application Loadbalancer
```

Comment:
typo: `which is extract` => `which extracts` maybe? also explain the use case (e.g. extract from what)